static/sass/_pattern_blog-featured.scss

```diff
@@ -0,0 +1,42 @@
+@mixin blog-p-featured {
+  $featured-bg-dark: #0d5777;
+  $featured-bg-light: #47929d;
+
+  .p-strip--featured {
```

Comment:
Could we use the Vanilla Suru strip pattern? https://vanillaframework.io/docs/patterns/strip#suru-strip you would just need to adjust the strip colour settings.

Reply:
Yeah, I was not using it.